src/Nginx/UX/Nginx.NginxPlus/nginxDeployments-certificates.json

```diff
@@ -1,13 +1,13 @@
 {
   "resourceType": "nginxDeployments/certificates",
-  "apiVersion": "2022-08-01",
+  "apiVersion": "2023-04-01",
   "learnMore": {
     "url": "https://learn.microsoft.com/powershell/module/az.nginx"
   },
   "commands": [
     {
       "name": "Get-AzNginxCertificate",
-      "description": "Get a certificate of given Nginx deployment",
```

Comment:
These are generated from swagger not modified manually.

Because "NGINX" is the word we want to use for all the customer facing messages. The swagger file is updated with "NGINX" in the description.